tasks/section5.yml

```diff
@@ -435,7 +435,11 @@
       - rule_5.5
 
 - name: "SCORED | 5.6 | PATCH | Ensure access to the su command is restricted"
-  command: /bin/true
+   lineinfile:
```

Comment:
This task is failing the syntax check. Looks like an extra space here in front of `lineinfile:`